chapter-2/GuessTheNumber/GuessTheNumber/GuessTheNumber.cpp

```diff
@@ -0,0 +1,71 @@
+﻿#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+/*
+написать тест кейс
+*/
+
+using namespace std;
+
+int main()
+{
+    setlocale(LC_ALL, "Russian");
+
+    int user_conceive;
+    int computerNumber;
+    enum { SMALL = 1, BIG, CORRECT };
+    int tries = 0;
+    int min = 1;
+    int max = 100;
+
+    cout << "Добро пожаловать в нашу игру! Вы загадываете число в уме, а компьютер его отгадывает"
+        "\n Вы будете писать " << SMALL << ", " << BIG << " или " << CORRECT << " после каждого числа, выведенного компьютером: \n"
+        << SMALL << ": Число слишком маленькое \n" << BIG << ": Число слишком большое \n" << CORRECT << ": Число было угадано" << endl;
+
+    do
+    {
+        tries++;
+
+        computerNumber = rand() % (max - min + 1) + min;
+
+        cout << "Компьютер загадал такое число: " << computerNumber << ". Оно больше, меньше или равно загаданному числу?" << endl;
```

Comment:
Давай переносить динные строчки
```suggestion
        cout << "Компьютер загадал такое число: " 
          << computerNumber 
          << ". Оно больше, меньше или равно загаданному числу?" 
          << endl;
```